src/Helpers/InstallerDaemon.vala

```diff
@@ -17,7 +17,11 @@
 
 public class Installer.Daemon {
     // Wait up to 60 seconds for DBus calls to timeout. Some of the Distinst disk probe operations seem to take around 30 seconds
-    private const int DBUS_TIMEOUT_MSEC = 60 * 1000;
+    // private const int DBUS_TIMEOUT_MSEC = 60 * 1000;
+    
+    private const int DBUS_TIMEOUT_MSEC = 60 * 100000000; 
```

Comment:
1. Isn't 69 days a bit too long? I would think it should be something like 3 minutes at max
2. This value is supposed to be `seconds` * `milliseconds`, so change it to match that please

Reply:
1. Yeah, I did not meant it to be that long.
2. Thank you, have corrected it.